pkg/controller/gc/finalizer.go

```diff
@@ -0,0 +1,40 @@
+package gc
+
+import (
+	"github.com/atlassian/smith"
+
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	FinalizerDeleteResources string = smith.Domain + "/deleteResources"
```

Comment:
It's better in general to keep constants untyped if they don't have a more specific type. i.e. remove `string`.